code/go/0chain.net/chaincore/chain/state.go

```diff
@@ -512,6 +512,15 @@ func (c *Chain) transferAmount(sctx bcstate.StateContextI, fromClient, toClient
 		}
 		return err
 	}
+
+	if err = c.emitEvent(sctx,stateToUser(fromClient, fs), event.TagAddOrOverwriteUser); err != nil {
```

Comment:
The `fromClient` should have already been in the db or the event has been emitted, if we do again here, we may encounter db insert error like 'user already exist'.